### PR TITLE
Ajay tripathy bc scaffolding

### PR DESCRIPTION
**Pull request by AjayTripathy**
## What does this PR change?
Rename files + controller name and add values yaml

## Does this PR rely on any other PRs?
No

## How does this PR impact users? (This is the kind of thing that goes in release notes!)
Use a more clear naming convention

## How was this PR tested?
Manually deployed files and saw results of admissions controller

## Have you made an update to documentation?
No;todo

### CODE REVIEW

File: cost-analyzer/templates/kubecost-admission-controller-service-template.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{{- if .Values.kubecostAdmissionController -}}
+{{- if .Values.kubecostAdmissionController.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-server
```
Comment (michaelmdresser) [inline cost-analyzer/templates/kubecost-admission-controller-service-template.yaml:6]: I assume we'll be revisiting this name in a future PR?

**Review by michaelmdresser — APPROVED** [review]
As far as I can tell, this is just moving the templates enabled by `budgetController.enabled=true` to templates enabled by `kubecostAdmissionController.enabled=true`. LGTM on that front.

Do we have any plans to remove the old templates?

File: cost-analyzer/templates/kubecost-admission-controller-service-template.yaml (hunk 1 above)
Comment (AjayTripathy) [reply cost-analyzer/templates/kubecost-admission-controller-service-template.yaml:6]: this is baked into the SSL key. I'd have to regenerate everything...so we can revisit in a future PR.

**Comment by AjayTripathy** [comment]
> As far as I can tell, this is just moving the templates enabled by `budgetController.enabled=true` to templates enabled by `kubecostAdmissionController.enabled=true`. LGTM on that front.
> 
> Do we have any plans to remove the old templates?

I don't see them in the source tree.. am I missing something?

**Comment by michaelmdresser** [comment]
> I don't see them in the source tree.. am I missing something?

https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/templates/budget-controller-template.yaml

**Comment by AjayTripathy** [comment]
https://github.com/kubecost/cost-analyzer-helm-chart/tree/AjayTripathy-bc-scaffolding/cost-analyzer/templates

Hahaha my bad. I guess I need to commit the git rm and not just move them here^

**Comment by AjayTripathy** [comment]
https://github.com/kubecost/cost-analyzer-helm-chart/pull/1935

**Pull request merged**